src/core/symbol/SymbolMessageUtils.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * The Message type. Supported supply types are:
+ * -1: RawMessage (no type appended)
+ * 0: PlainMessage
+ * 1: EncryptedMessage.
+ * 254: Persistent harvesting delegation.
+ */
+
+import { Converter } from '@utils';
+
+export enum MessageType {
+    PlainMessage = 0x00,
+    EncryptedMessage = 0x01,
+    PersistentHarvestingDelegationMessage = 0xfe,
+}
+
+// TODO add more methods to serialize and deserialize messages for each Message type.
```

Comment:
TODO*:*

Reply:
I'm thinking that the Message api (serialize and deserialize plain, raw, encrypted, and delegate messages ) is outside the scope of this PR. I can them if you like and discuss the API here. The Message API should be compatible with other SDKs so one SDK can read another SDK's message. I would generate vector tests for these.

One problem I found is that the "KeyPair" doesn't have a "encrypt" method to encrypt the crypto operations from the Message API. Should we add it? Or discuss this in another PR?

@Jaguar0625 @gimer

Reply:
I have created this issue, please have a look and let me know if it makes sense

https://github.com/symbol/sdk-typescript/issues/40

Reply:
I have removed SymbolMessageUtils.ts , we can add a class similar if required when working on #40